Resources/Private/Templates/Standard/Index.html

```diff
@@ -1,165 +1,89 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE html
-     PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
-     "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<!DOCTYPE html>
+<html lang="en">
 <head>
-<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-<title>Welcome to Flow</title>
-<link rel="stylesheet" type="text/css" href="../../../Public/Styles/Styles.css" />
-<script type="text/javascript" src="../../../Public/Libraries/jquery/jquery-1.9.1.min.js"></script>
-<script type="text/javascript">
-$(document).ready(function() {
-	$('.tab').each(function(index, element) {
-		$(element).addClass(index == 0 ? 'tab-active' : 'tab-inactive');
-		$(element).find('h2').first().click(function() {
-			$('.tab').removeClass('tab-active').addClass('tab-inactive');
-			$(element).addClass('tab-active').removeClass('tab-inactive');
-		});
-	});
-});
-</script>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@100;300&display=swap" rel="stylesheet">
```

Comment:
Honestly I have concerns about including fonts from google. Would it be possible to add the font to the repo, or if not, link it cookie-free from neos.io?